### PR TITLE
Compact some compute_ctl logs

### PR DESCRIPTION
**Pull request by petuhovskiy**
Print postgres roles in a single line and add some info.

### CODE REVIEW

File: compute_tools/src/spec.rs [hunk 1]
```diff
@@ -239,7 +241,10 @@ pub fn handle_roles(spec: &ComputeSpec, client: &mut Client) -> Result<()> {
     // Refresh Postgres roles info to handle possible roles renaming
     let existing_roles: Vec<Role> = get_existing_roles(&mut xact)?;
 
-    info!("cluster spec roles:");
+    info!(
+        "handling cluster spec roles (total {}):",
```
Comment (petuhovskiy) [inline compute_tools/src/spec.rs:245]:
```suggestion
        "handling cluster spec roles (total {})",
```

File: compute_tools/src/spec.rs [hunk 2]
```diff
@@ -503,7 +509,10 @@ pub fn handle_databases(spec: &ComputeSpec, client: &mut Client) -> Result<()> {
     // Refresh Postgres databases info to handle possible renames
     let existing_dbs = get_existing_dbs(client)?;
 
-    info!("cluster spec databases:");
+    info!(
+        "handling cluster spec databases (total {}):",
```
Comment (petuhovskiy) [inline compute_tools/src/spec.rs:513]:
```suggestion
        "handling cluster spec databases (total {})",
```

File: compute_tools/src/spec.rs [hunk 3]
```diff
@@ -562,7 +571,7 @@ pub fn handle_databases(spec: &ComputeSpec, client: &mut Client) -> Result<()> {
                 DatabaseAction::Create => " -> create",
                 DatabaseAction::Update => " -> update",
             };
-            info!("   - {}:{}{}", db.name, db.owner, action_str);
+            info!(" - {}:{}{}", db.name, db.owner, action_str);
```
Comment (save-buffer) [inline compute_tools/src/spec.rs:574]: Maybe leave the two spaces in so it looks like an indented list? Or maybe add `\t`? Or should we accumulate all of these into a vector as well?

File: compute_tools/src/spec.rs [hunk 4]
```diff
@@ -319,7 +324,7 @@ pub fn handle_roles(spec: &ComputeSpec, client: &mut Client) -> Result<()> {
                 RoleAction::Create => " -> create",
                 RoleAction::Update => " -> update",
             };
-            info!("   - {}:{}{}", name, pwd, action_str);
+            info!(" - {}:{}{}", name, pwd, action_str);
```
Comment (save-buffer) [inline compute_tools/src/spec.rs:327]: Same as for databases

**Review by save-buffer — APPROVED** [review]
Makes sense, don't have any problems. Overall looks good, but had question about two parts

**Pull request merged**